ProjektCsharp2/DailyRecords.cs

```diff
@@ -52,39 +45,53 @@ public void AddWeight(WeightMonitoring weight)
             File.AppendAllText(WeightFile, $"{weight.Weight},{weight.Date:dd-MM-yyyy}\n");
         }
 
+        // Tady nam muzou vyskocit par vyjimek pri cteni a parsovani dat ze souboru, bylo by vhodne 
+        // je osetrit a vypsat uzivateli chybovou hlasku.
         public void LoadFile()
         {
             if (File.Exists(MealFile))
             {
                 //var lines = File.ReadAllLines(MealFile);
-                foreach (var line in File.ReadAllLines(MealFile).Skip(1))
+                foreach (var line in File.ReadAllLines(MealFile).Skip(1)) // Hodil by se nejaky komentar, proc preskakujeme prvni radek
                 {
                     var mealParts = line.Split(',');
                     //Meals.Add(new Meal { NameOfMeal = mealParts[0], Calories = int.Parse(mealParts[1]), Date = DateTime.Parse(mealParts[2]) });
                     if (mealParts.Length == 3)
                     {
-                        Meals.Add(new Meal { NameOfMeal = mealParts[0], Calories = int.Parse(mealParts[1]), Date = DateTime.Parse(mealParts[2]) });
+                        Meals.Add(new Meal
+                        {
+                            NameOfMeal = mealParts[0],
+                            Calories = int.Parse(mealParts[1]),
+                            Date = DateTime.ParseExact(mealParts[2], "dd-MM-yyyy", null)
```

Comment:
Konstruktor jsem rozdelil na nekolik radku pro citelnost, jinak se nam tady vloudil maly bug - `DateTimeParse` parsuje podle formatu uzivatele, takze se to muze menit podle pocitace, na kterem to spustime.